backend/lib/ludwig/api-possible-values.js

```diff
@@ -1,122 +1,38 @@
-        'id': 'apa_eligibilite',
-        'shortLabel': 'APA - Éligibilité',
-    },
-    {
-        'id':'aspa',
-        'shortLabel':'ASPA',
-    },
-    {
-        'id':'asi',
-        'shortLabel':'ASI',
-    },
-    {
-        'id':'af',
-        'shortLabel':'AF',
-    },
-    {
-        'id':'cf',
-        'shortLabel':'CF',
-    },
-    {
-        'id':'asf',
-        'shortLabel':'ASF',
-    },
-    {
-        'id':'paje_base',
-        'shortLabel':'PAJE-BASE',
-    },
-    {
-        'id':'rsa',
-        'shortLabel':'RSA',
-        'uncomputabilityReasons': {
-            'tns': 'le demandeur a des revenus en tant qu’indépendant',
-            'conjoint_tns': 'le conjoint du demandeur a des revenus en tant qu’indépendant'
-        }
-    },
-    {
-        'id':'aide_logement',
-        'shortLabel':'AL',
-        'uncomputabilityReasons': {
-            'primo_accedant': 'le demandeur est primo-accédant',
-            'locataire_foyer': 'le demandeur loge dans un foyer'
-        }
-    },
-    {
-        'id':'ass',
-        'shortLabel':'ASS',
-    },
-    {
-        'id':'bourse_college',
-        'shortLabel':'Bourse collège',
-    },
-    {
-        'id':'bourse_lycee',
-        'shortLabel':'Bourse lycée',
-    },
-    {
-        'id':'paris_logement_familles',
-        'shortLabel':'PLF'
-    },
-    {
-        'id': 'paris_forfait_famille',
-        'shortLabel': 'PFF'
-    },
-    {
-        'id': 'paris_logement_psol',
-        'shortLabel': 'PSOL'
-    },
-    {
-        'id': 'paris_logement',
-        'shortLabel': 'PL'
-    },
-    {
-        'id': 'paris_logement_aspeh',
-        'shortLabel': 'ASPEH'
-    },
-    {
-        'id': 'paris_logement_plfm',
-        'shortLabel': 'PLFM'
-    },
-    {
-        'id': 'paris_energie_famille',
-        'shortLabel': 'PEF'
-    },
-    {
-        'id': 'paris_complement_sante',
-        'shortLabel': 'CSP'
-    },
-    {
-        'id': 'aah',
-        'shortLabel': 'AAH',
-    },
-    {
-        'id': 'caah',
-        'shortLabel': 'CAAH',
-    },
-    {
-        'id': 'ppa',
-        'shortLabel': 'PPA',
-    },
-    {
-        'id': 'rennes_metropole_transport',
-        'shortLabel': 'Rennes - Transport',
-        'unit': '%',
-    },
-    {
-        'id': 'brest_metropole_transport',
-        'shortLabel': 'Brest - Transport',
-        'unit': '%',
-    },
-];
+var droits = require('../../../app/js/constants/droits');
+
+var possibleValues = [];
+Object.keys(droits).forEach(function(prestationLevel) {
+    var providers = droits[prestationLevel];
+
+    Object.keys(providers).forEach(function(providerName) {
+        var provider = providers[providerName];
+
+        var options = Object.keys(provider.prestations).map(function(prestationName) {
+            var prestation = provider.prestations[prestationName];
+            var label = prestation.label == 'Tarification solidaire transports' ? (provider.label + ' - ' + prestation.label) : prestation.label;
```

Comment:
Can we avoid hardcoding this? I think it is ok to append the city name to all local benefits.

Reply:
Thanks, I refactored that section.